### DISCUSSION TITLE
how to implement videojs on a website

### DISCUSSION THREAD

**Discussion by huntergames79** [videojs/video.js#8006] [General]
Hello,
 Is there a simple tutorial on how to implement the video on a website from the very beginning, for none savvy programming person? 
 The youtube videos I have watched are very hard to comprehend. Thank's in advance!

**Comment by mister-ben** [comment]
There's the most basic implementation here to copy and paste and update with your video URL: https://videojs.com/getting-started/#videojs-cdn

**Comment by densen2014** [comment]
The official document is the simplest
